src/test/java/de/tum/in/www1/artemis/quiz/QuizExerciseIntegrationTest.java

```diff
@@ -1566,28 +1097,27 @@ void importQuizExerciseFromCourseToExam_forbidden() throws Exception {
     @Test
     @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
     void importQuizExerciseFromExamToCourse() throws Exception {
-        ExerciseGroup exerciseGroup1 = database.addExerciseGroupWithExamAndCourse(true);
-        quizExercise = database.createQuizForExam(exerciseGroup1);
+        QuizExercise quizExercise = database.createAndSaveExamQuiz(ZonedDateTime.now(), ZonedDateTime.now().plusDays(1));
+
+        quizExercise.setExerciseGroup(null);
         Course course1 = database.addEmptyCourse();
-        quizExerciseService.save(quizExercise);
         quizExercise.setCourse(course1);
-        quizExercise.setExerciseGroup(null);
 
-        request.postWithResponseBody("/api/quiz-exercises/import/" + quizExercise.getId(), quizExercise, QuizExercise.class, HttpStatus.CREATED);
+        QuizExercise importedExercise = request.postWithResponseBody("/api/quiz-exercises/import/" + quizExercise.getId(), quizExercise, QuizExercise.class, HttpStatus.CREATED);
+        assertThat(importedExercise.getCourseViaExerciseGroupOrCourseMember()).isEqualTo(course1);
     }
 
     /**
      * test import quiz exercise from exam to course with invalid roles
      */
     @Test
-    @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "TA")
+    @WithMockUser(username = TEST_PREFIX + "editor1", roles = "TA")
```

Comment:
```suggestion
    @WithMockUser(username = TEST_PREFIX + "tutor1", roles = "TA")
```

considering the role